azure-functions/acbs-function/package.json

```diff
@@ -29,7 +29,7 @@
     "unit-test-ff": "jest --coverage --verbose --config=unit.ff.jest.config.js --passWithNoTests"
   },
   "dependencies": {
-    "axios": "^1.7.8",
+    "axios": "1.7.8",
```

Comment:
dependabot may not listen to this, it did not on EXIPs but worth to see if it does